src/i18n/en.json

```diff
@@ -1454,7 +1454,18 @@
       "investment-submitted-msg": "Invested successfully. Your account will be funded with {{asset}} once the sale succeeds",
       "offer-canceled-msg": "Your offer was successfully cancelled",
       "requires-verification-title": "Verification required",
-      "requires-verification-desc": "Please, verify your account to invest in this sale"
+      "requires-verification-desc": "Please, verify your account to invest in this sale",
+      "deposit-btn": "Deposit",
+      "fees": "Fees",
```

Comment:
do not add unused translations

Reply:
please recheck the added translations and ensure there is no unused entries anymore

Reply:
rechecked, removed